introduction.tex

```diff
@@ -7,6 +19,14 @@ \section{Introduction}
 to receive all the \gls{SNF} from EU nations to create \gls{MOX}.
 The \gls{MOX} created will fuel French transition to a \gls{SFR} fleet
 and may allow France to avoid building additional \glspl{LWR}.
+
+Past research focuses only on France, where additional \glspl{LWR},
+namely \glspl{EPR} are constructed in order to supply \gls{SNF} required
+for \gls{MOX} production \cite{carre_overview_2009, martin_symbiotic_2007, freynet_multiobjective_2016}.
+There has been little attention in reprocessing legacy \gls{SNF} from other
+EU nations to produce \gls{MOX} for the newly deployed \glspl{SFR}.
+This rids the need to construct additional \glspl{LWR}.
```

Comment:
"This" has an ambiguous antecedent. Read literally, it sounds like you mean "this"="little attention". But, clearly, you mean the strategy. I recommend changing this sentence to "The present work finds that this collaborative strategy can reduce the need to construct additional LWRs in France."